pkg/stub/handler.go

```diff
@@ -430,24 +430,25 @@ func (h *Handler) tbrInaccessible() bool {
 		h.imageConfigBlocksImageStreamCreation("quay.io") {
 		return true
 	}
-	err := wait.PollImmediate(5*time.Second, 20*time.Second, func() (bool, error) {
+
+	proxy := &configv1.Proxy{}
+	var err error
+	wait.PollImmediate(5*time.Second, 20*time.Second, func() (bool, error) {
 		// if a proxy is in play, the registry.redhat.io connection attempt during startup is problematic at best;
 		// assume tbr is accessible since a proxy implies external access, and not disconnected
-		proxy, err := h.configclient.Proxies().Get(context.TODO(), "cluster", metav1.GetOptions{})
+		proxy, err = h.configclient.Proxies().Get(context.TODO(), "cluster", metav1.GetOptions{})
 		if err != nil {
 			logrus.Printf("unable to retrieve proxy configuration as part of testing registry.redhat.io connectivity: %s", err.Error())
 			return false, nil
-		} else {
-			if len(proxy.Status.HTTPSProxy) > 0 || len(proxy.Status.HTTPProxy) > 0 {
-				logrus.Printf("with global proxy configured assuming registry.redhat.io is accessible, bootstrap to Managed")
-				return true, nil
-			}
 		}
-		return false, errors.New("proxy is not configured")
+		return true, nil
 	})
-	if err != nil {
+
+	if len(proxy.Status.HTTPSProxy) > 0 || len(proxy.Status.HTTPProxy) > 0 {
```

Comment:
Is there any chance that `proxy.Status` could be nil here? Or are we confident that `h.configclient.Proxies.Get` will provide that even if there is an error?

Reply:
nope it is not a pointer ... no nil ref issues here